test/unit/org/apache/cassandra/cql3/validation/operations/AlterTest.java

```diff
@@ -821,8 +821,7 @@ public void testAlterTableAddColWithIfNotExists() throws Throwable
     public void testAlterTableAddExistingColumnWithoutIfExists()
     {
         createTable("CREATE TABLE %s (a int, b int, PRIMARY KEY (a, b)); ");
-        assertAlterTableThrowsException(InvalidRequestException.class,
-                                        String.format("Column with name '%s' already exists", "a"), "ALTER TABLE IF EXISTS %s ADD a int");
+        assertAlterTableThrowsException(InvalidRequestException.class, String.format("Column with name '%s' already exists", "a"), "ALTER TABLE IF EXISTS %s ADD a int");
```

Comment:
I was referring to the fact we try to avoid long lines. This would probably end up in 3 lines, one for each arg, and the 3 args aligned. Look [here](https://github.com/apache/cassandra/pull/1422/commits/46f617900d1f2061bff38440036b420216634816#diff-60155f5fd589ec7d5d5cba38462b429d211fa0c70a3c3283972dd6e865c1eebbR189) i.e. or the C* code style

It's a nit, but just to let you know this is how we format things.

Reply:
Sorry for the misunderstanding. I should have gone through the C* code style templates better. Will keep this in mind and review code-styles. Fixed it in the latest commit. Agree it a nit, but I think it's good to follow the correct way of doing it. Appreciate it.